### PR TITLE
Pause profiler before opening profile analysis page

### PR DESCRIPTION
**Pull request by schien**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -187,6 +191,11 @@ function collectProfile() {
     console.log("error getting profile:");
     console.log(error)
     tabOpenPromise.then(tab => tab.url = `data:text/html,${error}`);
+  }).then(function () {
+    // Resume the profiler after profiler page is opened.
```
Comment (digitarald) [inline index.js:195]: Maybe that could be a pref (off by default)? It doesn't seem to make sense to continue profiling when you just captured a new profile. If you want to profile, you would usually start the profiler before reproducing an issue anyways.

Comment (schien) [reply index.js:195]: Sure that's also what I want but seems a different feature, may be open another issue?

**Comment by mstange** [comment]
Thanks schien! This is a good workaround until [bug 1321617](https://bugzilla.mozilla.org/show_bug.cgi?id=1321617) is fixed.

File: index.js (hunk 1 above)
Comment (mstange) [reply index.js:195]: It absolutely makes sense to continue profiling; having the profiler running in the background at all times is one of the main use cases we want to support. But sure, we can add a pref to not do that.

**Pull request merged**